frontend/src/components/SearchFiltersSidebar.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import "../style/App.css";
+import SearchTool from "./SearchTool";
 
 export default function SearchFiltersSidebar() {
-  return <div className="search-sidebar">Search Sidebar</div>;
+  return (<SearchTool />);
```

Comment:
Nice!  You can leave off the "()"

They are only needed for multi line return statements.